### DISCUSSION TITLE
not working on Macos

### DISCUSSION THREAD

**Discussion by noob91-s** [pear-devs/pear-desktop#248] [Q&A]
Im not sure why, but this message always appears when i try to open the youtube-music app.  
![image](https://user-images.githubusercontent.com/83655570/117020572-6a6f5b80-acab-11eb-8ffc-35270d9585c7.png) I I downloaded the zip file and the opened it, is there another step I skipped? Also is there a version for android? I would like to get this on my phone as well. Thank you for making this great app!

**Comment by Araxeus** [comment]
I personally don't have a mac but it sounds like some kind of permission issue, 
someone else should hopefully have an answer about that.

For Android you should probably check out [Vanced Music](https://vancedapp.com/), they are pretty great 🙂

**Reply by noob91-s** [reply]
thank you!!

**Reply by noob91-s** [reply]
do you know what permissions you have allowed for the app? I dont want to add permissions that the app wont need.

**Reply by Araxeus** [reply]
I don't have mac so I can only redirect you to stuff I found online about this problem 😅 
[Apple1](https://discussions.apple.com/thread/251711574) | [apple2](https://discussions.apple.com/thread/7586240) | [SuperUser](https://superuser.com/questions/1345755/how-to-fix-the-application-cant-be-opened-on-mac)

I hope one of those answers can help you

**Comment by semvis123** [comment]
Have you tried the .dmg version?
What macOS version are you on? it works fine for me on Big Sur.

**Comment by mccre110** [comment]
I had some issues after attempting to install with the `dmg` on Sequoia.

This is what worked for me:
`xattr -d com.apple.quarantine /Applications/YouTube\ Music.app`

[Source](https://github.com/deskflow/deskflow/wiki/Running-on-macOS)
